_includes/releases.html

```diff
@@ -2,7 +2,9 @@
 	<thead>
 		<tr>
 			<th>Version</th>
+			{% unless include.releases.first.prerelease %}
```

Comment:
Looks good, but let's just add a comment here describing that releases are either all prereleases or all not prereleases.

Reply:
Liquid comment (https://shopify.github.io/liquid/tags/template/#comment) or HTML comment?